decidim-elections/spec/shared/monitoring_committee_member_manages_voting_examples.rb

```diff
@@ -2,7 +2,7 @@
 
 shared_context "when monitoring committee member manages voting" do
   let(:organization) { create(:organization) }
-  let(:user) { create(:user, :confirmed, organization:) }
+  let(:user) { create(:user, :confirmed, admin_terms_accepted_at: Time.current, organization:) }
```

Comment:
You can also use the `:admin_terms_accepted` trait here.
```suggestion
  let(:user) { create(:user, :confirmed, :admin_terms_accepted, organization:) }
```